src/extension/utils/helpers.ts

```diff
@@ -30,3 +30,13 @@ export function compareSemanticVersion(left: string, right: string) {
 
   return 0;
 }
+
+export function isClassInChangedNodes(className: string, changedNodes: Set<string>) {
+  for (const node of changedNodes) {
+    if (!!node?.split(' ')?.some((c) => c === className)) {
+      return true;
+    }
+  }
+
+  return false;
+}
```

Comment:
hmm this is definitely cleaner -- it's going to be slightly less performant but _maybe_ not enough to worry about.

another option would just to do this split in the ObserverHandler so we could still do `.has(...)` in the features but this will work for now